translation/v3_to_v2.xsl

```diff
@@ -103,14 +99,10 @@ Version: 0.1.0
   <xsl:template match="auc:DoorIDs">
     <xsl:copy-of select="auc:DoorID"/>
   </xsl:template>
-  <!-- 5. Transform auc:eGRIDSubregionCodes/auc:eGRIDSubregionCode element to auc:eGRIDRegionCode element -->
-  <xsl:template match="auc:eGRIDSubregionCodes">
-    <xsl:copy-of select="auc:eGRIDRegionCode"/>
-  </xsl:template>
```

Comment:
What's the reason for removing this?

Reply:
We had the `eGRIDSubregionCodes` added in 2.4 as a `<xs:choice>` element in parallel with `eGRIDRegionCode`, so we don't need to change it back now.